src/rest-server/src/controllers/v2/group.js

```diff
@@ -104,6 +104,29 @@ const updateGroupExtension = async (req, res, next) => {
   }
 };
 
+const updateGroupExtensionAttr = async (req, res, next) => {
+  try {
+    const groupname = req.params.groupname;
+    const attrs = req.params[0].split('/');
+    const updateData = req.body.data;
```

Comment:
The same problem. We have no idea about this data.

Reply:
Because we don't know the updated attrs or its type, here I use data as a general name.